package.json

```
@@ -3,14 +3,16 @@
   "private": true,
   "scripts": {
     "lint": "remark --frail .",
-    "lint:fix": "remark --output --frail ."
+    "lint:fix": "remark --output --frail .",
+    "check_prs": "node ./utils/check_prs.js"
```

```suggestion
    "check-prs": "node ./utils/check-prs.js"
```

It's confentional to use dashes and camelCase for JavaScript.